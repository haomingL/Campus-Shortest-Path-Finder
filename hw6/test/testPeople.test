LoadGraph people people.tsv
FindPath people Xinyi-Xu Haoming-Liu
FindPath people Haoming-Liu Chenfan-Sun
FindPath people Youying-Lin Chenfan-Sun
FindPath people Haoming-Liu Haoming-Liu
FindPath people Shuo-Yin Haoming-Liu
FindPath people Haoming-Liu Ziqiu-Zhang