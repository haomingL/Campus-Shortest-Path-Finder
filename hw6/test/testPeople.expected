loaded graph people
path from Xinyi-Xu to Haoming-Liu:
Xinyi-Xu to Haoming-Liu via CSE331
path from Haoming-Liu to Chenfan-Sun:
Haoming-Liu to Yingkai-Wang via CSE331
Yingkai-Wang to Chenfan-Sun via CSE390
path from Youying-Lin to Chenfan-Sun:
Youying-Lin to Yingkai-Wang via CSE331
Yingkai-Wang to Chenfan-Sun via CSE390
path from Haoming-Liu to Haoming-Liu:
unknown character Shuo-Yin
unknown character Ziqiu-Zhang